library/src/test/java/ch/decent/sdk/api/java/ApiTest.java

```diff
@@ -154,15 +165,16 @@ public void OperationsHelperTest() {
         api.getOperationsHelper().transfer(credentials, "reference", new AssetAmount(10), "public memo", false);
         api.getOperationsHelper().transfer(credentials, "reference", new AssetAmount(10), "public memo", false, fee);
```

Comment:
so what happened to these methods? Why have they been removed?

Reply:
Because we are moving all ops into their respected api categories